colander_tools/mapping.py

```diff
@@ -83,7 +83,7 @@ class OpenMapping(colander.Mapping):
     A mapping where the keys are free-form and the values a specific type.
     """
 
-    def _impl(self, node, value, callback):
+    def _impl(self, node, value, callback, default_or_missing):
```

Comment:
This will break support for old colander versions, so it'd be a bit better if this took a default argument whose value is a sentinel object:
```
unset = object()

class ..:
    def _impl(self, node, value, callback, default_or_missing=unset)
```
along with a condition to only pass the extra argument if it has been passed in from the caller.